src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -255,6 +257,9 @@ bool Replayer<I>::get_replay_status(std::string* description,
   root_obj["replay_state"] = replay_state;
   root_obj["remote_snapshot_timestamp"] = remote_snap_info->timestamp.sec();
 
+  m_perf_counters->set(l_rbd_mirror_snapshot_remote_timestamp,
+                       remote_snap_info->timestamp.sec());
```

Comment:
Need to guard against `m_perf_counters` being NULL:
```suggestion
  if (m_perf_counters) {
    m_perf_counters->tset(l_rbd_mirror_snapshot_remote_timestamp,
                          remote_snap_info->timestamp);
  }
```